tests/interface_dag_elements/test_automatically_added_functions.py

```diff
@@ -329,18 +93,18 @@ def test_should_overwrite_functions_with_data_cols_that_only_differ_in_time_peri
 
 class TestCreateFunctionForTimeUnit:
```

Comment:
Can you make an issue to have some AI tool help us convert all tests to pytest-style?